README.md

```diff
@@ -104,7 +105,7 @@ You're a good person and decide to validate your HTML. You copy and paste the co
 The validator throws errors at you! Why? Assuming you haven't made any mistakes in your code, how could you go about accurately validating your HTML?
 
 ```
-Your answer...
+See what appears on the index.html.erb page? Doesn't artist.name need a #{} surrounding it?
 ```
```

Comment:
The validator throws errors because it doesn't like ERB tags. That being said, you can take a look at the source code in your browser (i.e., after the ERB has been translated into pure HTML) and paste that into the validator.